File: Programming-languages-and-laboratory/Java/Exercises/35_compito_java_2019_11_20/Hockey.java


public class Hockey extends Squadra {

    private double gol, falli;

    public Hockey(int codice, String sport, String nome, int vinte, int perse, double gol, double falli) {
        super(codice, sport, nome, vinte, perse);
        this.gol = gol;
        this.falli = falli;
    }

    public String toString() {
        return nome + "\t" + codice + "\t" + vinte + "\t" + perse + "\t" + gol + "\t" + falli + "\t-\t" + sport;
    }
}

File: II - Linguaggi di programmazione e laboratorio/Python/Exercises/35_compito_java_20191120/Hockey.py
from Squadra import Squadra


class Hockey(Squadra):
    def __init__(self, codice, sport, nome, vinte, perse, goal, falli):
        super().__init__(codice, sport, nome, vinte, perse, goal)
        self._falli = falli

    def __str__(self):
        return super().__str__() + "\t" + str(self._goal) + "\t" + str(self._falli) + "\t-\t" + self._sport
